include/boost/geometry/extensions/gis/io/wkb/write_wkb.hpp

```diff
@@ -25,7 +25,7 @@ template <typename G>
 struct write_wkb<point_tag, G>
 {
     template <typename OutputIterator>
-    static inline bool write(const G& geometry, OutputIterator& iter, 
+    static inline bool write(const G& geometry, OutputIterator iter,
```

Comment:
Here and below.
Why the `OutputIterator` is passed by value rather then by reference as in the original code?
I prefer the reference approach, as with the value approach it is not possible to recover the iterator value after the call to this method.